ivy_tests/test_ivy/test_container.py

```diff
@@ -532,54 +529,6 @@ def test_container_structural_diff(device):
     assert container_diff_same_only.to_dict() == container_diff.to_dict()
 
 
-def test_container_from_dict(device):
```

Comment:
I think we should keep this test, it basically tests the constructor `__init__` of the container when a `dict` is passed.